deployer/scripts/install.sh

```diff
@@ -373,6 +373,10 @@ function generate_objects() {
   generate_kibana
   generate_curator
   generate_fluentd
+
```

Comment:
can we get rid of `oc new-app logging-imagestream-template` above as well? the imagestreams are no longer necessary.